### PR TITLE
Add query for total superfluid assets, add other clis for superfluid

### PR DESCRIPTION
**Pull request by mattverse**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: proto/osmosis/superfluid/query.proto [hunk 1]
```diff
@@ -137,6 +141,18 @@ message ConnectedIntermediaryAccountResponse {
   SuperfluidIntermediaryAccountInfo account = 1;
 }
 
+message TotalSuperfluidDelegationsRequest {
+}
+
+message TotalSuperfluidDelegationsResponse {
```
Comment (ValarDragon) [inline proto/osmosis/superfluid/query.proto:147]: Are the units in uosmo, if so can we add that as a commit?

File: proto/osmosis/superfluid/query.proto [hunk 2]
```diff
@@ -137,6 +141,18 @@ message ConnectedIntermediaryAccountResponse {
...
+  string totalDelgations = 1 [
```
Comment (ValarDragon) [inline proto/osmosis/superfluid/query.proto:148]:
```suggestion
  string totalDelegations = 1 [
```

**Review by ValarDragon — APPROVED** [review]
LGTM, I'll resolve my comments in a subsequent commit

**Comment by mattverse** [comment]
@ValarDragon Yeah the units are in uosmo, I think it might have been a better call to return it as `Coin`. Do you mind if I write a fix for it?

**Comment by ValarDragon** [comment]
Sure go for it

**Pull request merged**